draft-ietf-rtcweb-jsep.xml

```diff
@@ -5858,13 +5875,18 @@ a=msid:-
 
           <t>If the packet is of type SDES, and the sender SSRC for
           the packet is found in the incoming SSRC table, deliver the
-          packet to the "m=" line associated with that SSRC. In
-          addition, for each chunk in the packet that contains a MID
-          that is in the table mapping MID to "m=" line, update the
-          incoming SSRC mapping table to include an entry that maps
-          the SSRC for that chunk to the "m=" line associated with
-          that MID. (This case can occur when RTCP for a source is
-          received before any RTP packets.)</t>
+          packet to the "m=" line associated with that SSRC.
+          In addition, If that SDES packet contains SDES MID entries,
+          this can results in updates and additions to the incoming SSRC
+          to "m=" line mapping table. Thus each of the SDES MID items
+          are processed and the current table entries are checked if
+          the corresponding MID value matches the current SSRC
+          to "m=" line mapping, else the entry is updated. If there is
+          no RTP stream to "m=" line table mapping entry for the
+          received SDES item's SSRC, such an entry is created. Note,
+          that in the process of updating the table entries, update
+          flap suppression as discussed in Section 4.2.6 of <xref
+          target="RFC7941"/> needs to be considereds.</t>
```

Comment:
adding a reference to the flap issues seems OK but I this this actually needs some more text to discuss how this all works. I'd hate for the SSRC to change in a SFU, then the RTP sets the correct SSRC, then some sort of late RTCP packets messes up the mapping and we loose packets. This may actually point to not wanting to use RTCP for SDES.